src/index.html

```diff
@@ -31,7 +31,171 @@
     </nav>
   </header>
 
-  <main></main>
+  <main>
+    <div class="p-keyvisual">
+      <div class="l-inner">
+        <div class="p-keyvisual__titles">
+          <p class="p-keyvisual__keyvisual-text"><img src="./img/picture/keyvisual-text.png" alt="見出しバナー"></p>
+          <p class="p-keyvisual__keyvisual-title"><img src="./img/picture/keyvisual-title.png" alt="医療求人バナー"></p>
+           <div class="p-keyvisual__box">
+             <p>現在の求人掲載数：<span class="p-keyvisual__number">2,320</span>件</p>
+          </div>
+        </div>
+      </div>
+     </div>
+
+    <div class="p-jobsearch">
+        <hgroup>
+          <h2>お仕事検索</h2>
+          <p>働きたい職種・地域・希望給与を入れて検索！</p>
+        </hgroup>
+        <div class="p-jobsearch__select-box">
+          <div class="p-jobsearch__box_wrapper">
+            <select name="job" class="p-jobsearch__select">
+              <option value="job1" selected>医療事務</option>
+              <option value="job2">調剤事務</option>
+              <option value="job3">診療情報管理士</option>
+              <option value="job4">歯科助手</option>
+              <option value="job5">歯科事務</option>
+              <option value="job6">看護補助・看護助手</option>
+              <option value="job7">病院内SE</option>
+            </select>
+          </div>
+          <div class="p-jobsearch__box_wrapper">
+            <select name ="district" class="p-jobsearch__select">
+              <option value="district1">茨城県</option>
+              <option value="district2">神奈川県</option>
+              <option value="district3">群馬県</option>
+              <option value="district4">埼玉県</option>
+              <option value="district5">千葉県</option>
+              <option value="district6" selected>東京都</option>
+              <option value="district7">栃木県</option>
+            </select>
+          </div>
+          <div class="p-jobsearch__box_wrapper">
+            <form class="p-jobsearch__fome-box">
+            <select name="salary" class="p-jobsearch__select">
+              <option value="salary1" selected>月給17万〜</option>
+              <option value="salary2">月給18万〜</option>
+              <option value="salary3">月給19万〜</option>
+              <option value="salary4">月給20万〜</option>
+              <option value="salary5">月給21万〜</option>
+              <option value="salary6">月給22万〜</option>
+              <option value="salary">月給23万〜</option>
+            </select>
+            </form>
+          </div>
+          <input type="text" placeholder="キーワード・企業名・スキルなど" id="select-null" class="p-jobsearch__select-null">
+            <button class="form-button"><img src="./img/picture/glass.png" alt="虫眼鏡バナー">検索する</button>
+        </div>
+      </div>
+    <div class="p-joblistings">
```

Comment:
セクションの区切りなので、空行あけた方が見やすいかと思います。

```suggestion

    <div class="p-joblistings">
```

Reply:
間隔を修正致しました。
03ddc8ed50a1cb0442ceec5c760bd652e9964f1c